src/GraphClient.ts

```diff
@@ -0,0 +1,75 @@
+import ApolloClient from 'apollo-client';
+import { WebSocketLink } from 'apollo-link-ws';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { SubscriptionClient } from 'subscriptions-transport-ws';
+import gql from 'graphql-tag';
+
+import WebSocket = require('ws');
+
+/**
+ * The class interacts with graph node server for subscription and query.
+ */
+export default class GraphClient {
+  private subgraphEndPoint: string;
```

Comment:
Why is this private, this can be just read-only property.

Reply:
In the updated code apolloClient is the private variable which is not needed outside as per current use cases.